config/config/common-config.yaml

```diff
@@ -834,5 +834,9 @@ networkPolicies:
   dnsAutoscaler:
     enabled: true
 
+# Gatekeeper PSP for rook-ceph enabled/disabled
+rookCeph:
```

Comment:
we should enable this we we run init, if the cloud provider is exoscale
similar with what we do [here](https://github.com/elastisys/compliantkubernetes-apps/blob/main/bin/init.bash#L152)

Reply:
Added this